clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/converters/KubernetesAtomicOperationConverterHelper.groovy

```diff
@@ -24,17 +28,23 @@ class KubernetesAtomicOperationConverterHelper {
                                    AbstractAtomicOperationsCredentialsSupport credentialsSupport,
                                    Class targetDescriptionType) {
     def account = input.account as String
-    input.remove('credentials')
+    def optAccount = input.remove('credentials')
```

Comment:
If so, maybe `removedAccount` or `deletedAccount` would be clearer. I can live with it how it is though.